vmdb/lib/miq_automation_engine/service_models/miq_ae_service_orchestration_stack.rb

```diff
@@ -28,5 +28,13 @@ def remove_from_vmdb
       @object = nil
       true
     end
+
+    def raw_exists?
+      status, reason = raw_status
+      status.nil? || status.downcase == 'delete_complete' ? false : true
+    rescue => err
+      return false if err.to_s =~ /[S|s]tack.+does not exist/
```

Comment:
This error message was extracted from http respond 404 error. Can we solely depend on 404 without parsing the error message?